src/templates/blog-post.js

```diff
@@ -30,7 +30,7 @@ const BlogPostTemplate = (props) => {
       <SEO
         title={post.frontmatter.title}
         description={post.frontmatter.description || post.excerpt}
-        unicornData={post.frontmatter.author}
+        unicornData={post.frontmatter.authors} //might need to do CSV list here
```

Comment:
We don't want to do this. If we did, we would lose lots of what makes our GraphQL server so nice to use

Reply:
I guess I'm just not quite sure how we could otherwise present the list of authors.